### PR TITLE
Feature/repos projects organizations

### PR DESCRIPTION
**Pull request by xtuchyna**
## This introduces a breaking change

- [ ] Yes
- [ X ] No

## Description

There was a major misunderstanding of what project means in cli.
Now it is implemented to be valid with GitHub's terminology, that means:
- option project was changed to option *repository*
- option organization added to cli -> this allows to only specify the name of the organisation, from which all of the repositories will be inspected

### CODE REVIEW

File: srcopsmetrics/cli.py [hunk 1]
```diff
@@ -25,18 +25,28 @@
 from srcopsmetrics.bot_knowledge import visualize_project_results
 from srcopsmetrics.evaluate_scores import ReviewerAssigner
 
+from github import Github
 
 _LOGGER = logging.getLogger("aicoe-src-ops-metrics")
 logging.basicConfig(level=logging.INFO)
 
+_GITHUB_ACCESS_TOKEN = os.getenv("GITHUB_ACCESS_TOKEN")
```
Comment (pacospace) [inline srcopsmetrics/cli.py]: Isn't it already present in `GitHubKnowledge` class?

Comment (xtuchyna) [reply srcopsmetrics/cli.py]: I've added get_repositories function to GitHubKnowledge. Does it make sense now?

Comment (pacospace) [reply srcopsmetrics/cli.py]: Yes great!

File: srcopsmetrics/cli.py [hunk 2]
```diff
@@ -60,25 +70,27 @@
...
         analyse_projects(
-            projects=[repo.split("/") for repo in projects],
+            projects=[repo.split("/") for repo in repos],
             is_local=is_local
         )
 
     if visualize_statistics:
-        visualize_project_results(project=project, is_local=is_local)
+        for project in repos:
+            visualize_project_results(project=project, is_local=is_local)
```
Comment (pacospace) [inline srcopsmetrics/cli.py]: Also reviewer reccomender require:
```
for project in repos:
    reviewer_assigner = ReviewerAssigner()
    reviewer_assigner.evaluate_reviewers_scores(project=project, is_local=is_local)
```

File: srcopsmetrics/utils.py [hunk 3]
```diff
@@ -22,8 +22,10 @@
 
 import numpy as np
 
-from typing import Tuple
+from typing import Tuple, List
 from pathlib import Path
+from github import Github
+from github_knowledge import _GITHUB_ACCESS_TOKEN
```
Comment (pacospace) [inline srcopsmetrics/utils.py]: Are these required?

File: srcopsmetrics/github_knowledge.py [hunk 4]
```diff
@@ -72,6 +72,28 @@ def connect_to_source(self, project: Tuple[str, str]) -> Repository:
 
         return repo
 
+    @staticmethod
+    def get_repositories(repository: str = None, organization: str = None) -> List[str]:
```
Comment (pacospace) [inline srcopsmetrics/github_knowledge.py]: We should have a warning if no repos has been identified, WDYT?

Comment (xtuchyna) [reply srcopsmetrics/github_knowledge.py]: we should have warning if organization contains zero repositories
one repository would always be found, if not - an exception from pygithub occurs that repository does not exist
hence the warning should be only for organizations I think

File: srcopsmetrics/cli.py (hunk 2 above)
Comment (xtuchyna) [reply srcopsmetrics/cli.py]: gotme, cleared

File: srcopsmetrics/utils.py (hunk 3 above)
Comment (xtuchyna) [reply srcopsmetrics/utils.py]: gotme, cleared

File: srcopsmetrics/cli.py (hunk 2 above)
Comment (xtuchyna) [reply srcopsmetrics/cli.py]: done

File: srcopsmetrics/github_knowledge.py (hunk 4 above)
Comment (pacospace) [reply srcopsmetrics/github_knowledge.py]: Perfect! Thanks :)

File: srcopsmetrics/utils.py [hunk 5]
```diff
@@ -22,9 +22,10 @@
 
 import numpy as np
 
-from typing import Tuple
+from typing import Tuple, List
```
Comment (pacospace) [inline srcopsmetrics/utils.py]: Why these changes here? :)

File: srcopsmetrics/cli.py [hunk 6]
```diff
@@ -23,21 +23,31 @@
...
 from srcopsmetrics.bot_knowledge import visualize_project_results
+from srcopsmetrics.github_knowledge import GitHubKnowledge
 from srcopsmetrics.evaluate_scores import ReviewerAssigner
 
+from github import Github
 
 _LOGGER = logging.getLogger("aicoe-src-ops-metrics")
 logging.basicConfig(level=logging.INFO)
 
+_GITHUB_ACCESS_TOKEN = os.getenv("GITHUB_ACCESS_TOKEN")
```
Comment (pacospace) [inline srcopsmetrics/cli.py]: These are no more required here I guess :)

Comment (xtuchyna) [reply srcopsmetrics/cli.py]: jeesh, i'm so blind! should be more careful for small things like this!

File: srcopsmetrics/utils.py (hunk 5 above)
Comment (xtuchyna) [reply srcopsmetrics/utils.py]: removed

File: srcopsmetrics/utils.py [hunk 7]
```diff
@@ -25,6 +25,7 @@
 from typing import Tuple
 from pathlib import Path
 
+
```
Comment (pacospace) [inline srcopsmetrics/utils.py]: and this last line? :)
